src/Microsoft.VisualStudio.Services.Agent/Util/PowerShellExeUtil.cs

```diff
@@ -0,0 +1,121 @@
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.VisualStudio.Services.Agent.Util
+{
+    [ServiceLocator(Default = typeof(PowerShellExeUtil))]
+    public interface IPowerShellExeUtil : IAgentService
+    {
+        PowerShellInfo GetInfo();
```

Comment:
Ha! You're right! Now that the min version is validated by this class itself, we don't need to expose version anymore.